nemo/constants.py

```diff
@@ -12,41 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-import numpy as np
-
-# Supported Numpy DTypes: `np.sctypes`
-ACCEPTED_INT_NUMBER_FORMATS = (
```

Comment:
why is this gone? Was it used before?

Reply:
I don't think they are used. They were copied over when logger was refactored